types/winattr/index.d.ts

```diff
@@ -0,0 +1,21 @@
+export type Attr = "archive" | "hidden" | "readonly" | "system";
+
+export type Attrs = Record<Attr, boolean>;
+
+export function get(
+    path: string,
+    callback: <TError extends Error | null>(error: TError, attrs: TError extends TError ? never : Attrs) => void,
+): void;
+export function getSync(path: string): Attrs;
+
+export function set(path: string, attrs: Partial<Attrs>, callback: (error: Error | null) => void): void;
+export function setSync(path: string, attrs: Partial<Attrs>): void;
+
+declare const winattr: {
+    get: typeof get;
+    set: typeof set;
+    getSync: typeof getSync;
+    setSync: typeof setSync;
+};
+
+export default winattr;
```

Comment:
[Looking at the source](https://unpkg.com/winattr@3.0.0/lib/index.js), it uses the normal commonjs `module.exports` pattern, so you don't need anything but normal `export function`.

Reply:
Changes made